### DISCUSSION TITLE
Potential Vercel.com Google Indexing Issues

### DISCUSSION THREAD

**Discussion by awareness481** [vercel/next.js#13383] [Help] [answered]
I think I might have found an issue with http://vercel.com but I couldn't find any appropriate place to report it!

I was trying to find the http://vercel.com/design page **via google** as I couldn't remember the domain but I couldn't find it, so I tried the following google searches:

* site:http://vercel.com (Many results)
* site:http://vercel.com/design (0 Results)
* "design" site:http://vercel.com (0 Results)

*Quotes ("") around a keyword mean must include.*

Also tried

site:http://vercel.com/home (0 Results)

It seems like some pages on vercel.com aren't indexed by Google which seems to be unintentional. Pages like vercel.com/docs are indexed but pages like vercel.com/home or vercel.com/design aren't.

#### Reproducing

Use google with the following keywords:

* site:http://vercel.com/design
* "design" site:http://vercel.com
* site:http://vercel.com/home

*Expected* to see results for the relevant pages but got 0 results.

**Comment by c0b41** [comment]
vercel domain new, not fully indexed right now thats why.

**Comment by Timer** [comment, marked as answer]
Our design system is set to not index (intentionally), you'll see this in the HTML:

```html
<meta name="robots" content="noindex"/>
```

Thanks for checking!

**Reply by mrousavy** [reply]
I know this is pretty old, but out of curiosity - what's the reason for not indexing that?
